source/api_documentation/network_integration/ringpools/index.rst

```diff
@@ -94,6 +94,18 @@ Endpoint:
    :page: delete_ring_pool
 
 
+.. raw:: html
+
+  <br/>
+  <br/>
+  <div class="alert alert-info">
+    <b>NOTE: Creating Destinations</b></br>
+    If a valid phone number is given for destination_phone_number is given as part of the POST request, a Destination, aka Customer Phone Number, will be generated and associated with the created RingPool.<br/>
```

Comment:
🔧 `If a valid phone number is given for destination_phone_number is given as part of the POST request` -> `If a valid phone number is given for destination_phone_number as part of the POST request`